ci/build/engine_image_ref.go

```diff
@@ -0,0 +1,28 @@
+package build
+
+type EngineImageRef struct {
+	Wolfi   bool
+	GPU     bool
+	Image   string
+	Version string
+}
+
+func (e *EngineImageRef) String(explicitVersion string) string {
```

Comment:
`String` is usually indicated to return the string representation of a struct.

Not sure I understand why we have an `explicitVersion` here, it's quite confusing to follow.

Reply:
Yes, `EngineImageRef` is a struct and this is the string representation for it.

When `explicitVersion` is set, it overrides struct's `Version` property.